src/index.js

```diff
@@ -1,9 +1,73 @@
 'use strict';
 
-const defaultOptions = {};
+const Matrix = require('ml-matrix');
+const Algebra = Matrix.algebra;
```

Comment:
I didn't know that. Found that it is added by the curve-fitting package: https://github.com/mljs/curve-fitting/blob/master/src/index.js
This is very bad practice and should be removed. BTW do we still need to include curve-fitting now that levenberg-marquardt is here?